### DISCUSSION TITLE
missing eldap plugin in directory

### DISCUSSION THREAD

**Discussion by spham** [rabbitmq/rabbitmq-server#3131] [Other] [answered]
hi

```
sudo rabbitmq-plugins enable eldapEnabling plugins on node rabbit@adrmq01d:
eldap
Error:
{:plugins_not_found, [:eldap]}
```

ls /usr/lib/rabbitmq/lib/rabbitmq_server-3.8.17/plugins | grep eldap

not return any eldap plug in

test on redhat 7.9

**Comment by michaelklishin** [comment]
I will convert this issue to a GitHub discussion. Currently GitHub will [automatically close and lock the issue](https://github.com/github/feedback/discussions/3197) even though your question will be transferred and responded to elsewhere. This is to let you know that we do not intend to ignore this but this is how the current GitHub conversion mechanism makes it seem for the users :(

**Comment by michaelklishin** [comment, marked as answer]
`eldap` is not a plugin, it's a part of the Erlang/OTP standard library. You must be running on Debian (which slices and dices Erlang into dozens of packages) and do not have the [`erlang-eldap` package](https://packages.debian.org/buster/erlang-eldap) installed. This is documented in the [LDAP support guide](https://www.rabbitmq.com/ldap.html#prerequisites).
